source/adapter.js

```diff
@@ -278,7 +288,7 @@ AdapterJS.addEvent = function(elem, evnt, func) {
   }
 };
 
-AdapterJS.renderNotificationBar = function (text, buttonText, buttonLink) {
+AdapterJS.renderNotificationBar = function (text, buttonText, buttonLink, openNewTab, showRefreshButton) {
```

Comment:
Could've used "javascript:location.reload();" as buttonLink text. Than you don't need a separate "showRefreshButton" ;)